app/src/main/java/com/duckduckgo/app/tabs/db/TabsDao.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2018 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.tabs.db
+
+import android.arch.lifecycle.LiveData
+import android.arch.persistence.room.*
+import com.duckduckgo.app.tabs.model.TabEntity
+import com.duckduckgo.app.tabs.model.TabSelectionEntity
+import javax.inject.Singleton
+
+@Dao
+@Singleton
+abstract class TabsDao {
+
+    @Query("select * from tabs limit 1")
+    abstract fun firstTab(): TabEntity?
```

Comment:
optional: this implies there's an order to the tabs... maybe `anyTab()` would be a better name

Reply:
There is an order. They are maintained in the order they were created.

Reply:
Is that guaranteed by Room or somewhere else?

Because usually with an RDMBS you can't rely on that - if it happens to be the case it's an implementation detail of the database. 

If you want to guarantee order you need to use an `order by` statement and if you want to guarantee order by creation date, you need to insert that date when you create the row and sort by it.

Reply:
Good point - perhaps we do want a creation date + `order by`. thoughts @subsymbolic ?

Reply:
Great thinking! We discussed this in standup and I'll add a creation date as a separate PR.